src/app/curated-rolls/curatedRollService.ts

```diff
@@ -0,0 +1,167 @@
+import { DimStore } from '../inventory/store-types';
+import { toCuratedRolls } from './curatedRollReader';
```

Comment:
We've been moving towards kebab-case for non-react files, so `curated-roll-reader`.